db/migrate/20221116104013_add_capabilities_to_physical_storage_models.rb

```diff
@@ -1,8 +1,8 @@
 class AddCapabilitiesToPhysicalStorageModels < ActiveRecord::Migration[6.1]
   def change
-    add_column :physical_storage_families, :capabilities, :jsonb, :default => {}
-    add_column :physical_storages, :capabilities, :jsonb, :default => {}
-    add_column :storage_resources, :capabilities, :jsonb, :default => {}
-    add_column :storage_services, :capabilities, :jsonb, :default => {}
+    add_column :physical_storage_families, :capabilities, :jsonb, :default => []
+    add_column :physical_storages, :capabilities, :jsonb, :default => []
+    add_column :storage_resources, :capabilities, :jsonb, :default => []
+    add_column :storage_services, :capabilities, :jsonb, :default => []
```

Comment:
Since developers likely have already gotten past this, we should create a new migration